### PR TITLE
[skip ci] VIC 1.2 OVA Upgrade test

### LINKED ISSUE #5687: VIC 1.2 upgrade from 1.1 test
User statement:  As a customer of VIC, I want to be assured through testing that upgrading my existing VIC 1.1 install to VIC 1.2 will go smoothly and I will not lose any data.

Details:
https://gist.github.com/andrewtchin/7a6b24097147927ca97370fce53a2bce

Acceptance criteria:
VIC 1.2 upgrade process is documented in test plan and either manually executed or automated

### PR DESCRIPTION
**Pull request by singhshwetaQE**
Fixes #5687

### CODE REVIEW

File: tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+[VIC 1.2 Upgrade](TBD - waiting on official docs to link)
+
+# Environment:
+* This test requires that a vCenter server is running and available.
+* Existing vSphere Integrated Containers 1.1.x appliance that has deployed by using the official OVA installer.
+
+# Test Steps:
+1. Deploy new 1.2 OVA of the vSphere Integrated Containers appliance to a location in which it can access the VMDK files of the previous appliance.
+2. Log in to a vSphere Web Client instance to access both versions of the vSphere Integrated Containers appliance.
+3. Shut down both of the previous and new vSphere Integrated Containers appliances by selecting Shut Down Guest OS.
```
Comment (mhagen-vmware) [inline tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md]: Shut down both ~of~ the previous...

File: tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+4. Click on the previous vSphere Integrated Containers appliance, and Remove data disk from VM
+5. Click on the new vSphere Integrated Containers appliance.
+6. Remove and delete empty data disk from new OVA VM
+7. Attach old data disk to new OVA VM
+8. Power on the new appliance to complete the upgrade
+9. Verify that the data from your previous vSphere Integrated Containers installation has migrated successfully
```
Comment (mhagen-vmware) [inline tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md]: We need to have a mix of containers on the original VIC appliance and then we need to interact with those containers after upgrade, including creating some new ones.

File: tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md (hunk 1 above)
Comment (mdubya66) [inline tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md]: Upgrade from 1.1.1 as well.

File: tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md (hunk 2 above)
Comment (mdubya66) [inline tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md]: is this still valid with the changes coming from the harbor and admiral teams to help make this easier? @andrewtchin

Comment (andrewtchin) [inline tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md]: yes this is valid. there should also be a step after step 8 to run the upgrade script (that doesn't exist yet) to migrate/upgrade harbor and admiral

File: tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md (hunk 1 above)
Comment (singhshwetaQE) [inline tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md]: @mdubya66 I have included upgrade from 1.1.1

File: tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+# Purpose:
+To verify VIC 1.2 upgrade from 1.1.0 or 1.1.1 works and provides access to all of the vSphere Integrated Containers components.
+
+# References:
+[VIC 1.2 Upgrade](TBD - waiting on official docs to link)
+
+# Environment:
+* This test requires that a vCenter server is running and available.
+* Existing vSphere Integrated Containers 1.1.x appliance that has deployed by using the official OVA installer.
+* Deploy a variety of containers through command line and admiral
```
Comment (andrewtchin) [inline tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md]: Maybe call out that a VCH needs to be created before this?

File: tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+2. Log in to a vSphere Web Client instance to access both versions of the vSphere Integrated Containers appliance.
+3. Shut down both the previous and new vSphere Integrated Containers appliances by selecting Shut Down Guest OS.
+4. Click on the previous vSphere Integrated Containers appliance, and Remove data disk from VM
+5. Click on the new vSphere Integrated Containers appliance.
+6. Remove and delete empty data disk from new OVA VM
+7. Attach old data disk to new OVA VM
+8. Power on the new appliance to complete the upgrade
+9. Run the upgrade script to migrate harbor and admiral
+10. Verify that the data from your previous vSphere Integrated Containers installation has migrated successfully
+11. Verify that containers from previous VIC appliance exists after upgrade and run docker commands.
```
Comment (andrewtchin) [inline tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md]: Previous VCH should also exist

File: tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+12. Deploy a variety of containers through command line and admiral on new VIC appliance.
+13. Delete VCH
+14. Delete the OVA VM
```
Comment (andrewtchin) [inline tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md]: Delete old OVA VM

Comment (singhshwetaQE) [inline tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md]: done

**Pull request merged**